### PR TITLE
Added tipi.build support to transwarp

### PR DESCRIPTION
**Pull request by pysco68**
Hello, Yannic from [tipi.build](https://tipi.build) here,

I just added `tipi` support to your project and was wondering if you'd like to merge so everyone gets a new way of using your project. A customer had us try if tipi works on your code base during a demo so I thought we can contribute back (the build with msvc requires a `-bigobj` to get going ;)) 

If there's any issue with this I'll be happy to sort it out

Best from Zürich,
Yannic

### CODE REVIEW

File: .tipi/opts.vs-16-2019-win64-cxx17 [hunk 1]
```diff
@@ -0,0 +1 @@
+add_compile_options(-bigobj)
```
Comment (bloomen) [inline .tipi/opts.vs-16-2019-win64-cxx17:1]: What happens when the user wants to build with C++11?

File: README.md [hunk 2]
```diff
@@ -391,6 +391,16 @@ auto result = task->get(); // The final task's result remains valid
 The `releaser` also accepts an executor that gives control over _where_ a task's
 result is released.
 
+## Using transwarp with tipi.build
+
+`transwarp` can be easily used in [tipi.build](https://tipi.build) projects simply by adding the following entry to your `.tipi/deps`:
+
+```json
+{
+    "bloomen/transwarp": { }
```
Comment (bloomen) [inline README.md:400]: Where would transwarp be located or come from when integrated into a tipi project?

File: .tipi/opts.vs-16-2019-win64-cxx17 (hunk 1 above)
Comment (pysco68) [reply .tipi/opts.vs-16-2019-win64-cxx17:1]: Hi @bloomen

we're generally leaning torwards C++17 and later, so by default we currently don't ship a C++11 toolchain with tipi. If however someone wanted to compile for such a target they would have to provide an environment/toolchain-file and add 

```cmake
add_compile_definitions(TRANSWARP_CPP11)
```

...in their own project

File: README.md (hunk 2 above)
Comment (pysco68) [reply README.md:400]: We pull the sources straight from your github repository. Basically `bloomen/transwarp` is a shortcut for `github.com/bloomen/transwarp`. We currently default to pulling the latest revision (live at HEAD 💯 ) but it's easy to pin a version by adding an `@` section to the dependency configuration:

```json
{
    "bloomen/transwarp": { "@": "release-tag-ex-2.2.2" }
}
```

**Pull request merged**